podvm/qcow2/ubuntu/qemu-ubuntu.pkr.hcl

```diff
@@ -8,7 +14,7 @@ source "qemu" "ubuntu" {
   iso_checksum      = "${var.cloud_image_checksum}"
   iso_url           = "${var.cloud_image_url}"
   output_directory  = "${var.output_directory}"
-  qemuargs          = [["-device", "virtio-blk,drive=virtio-drive,id=virtio-disk0,bootindex=1"], ["-drive", "file=${var.output_directory}/${var.qemu_image_name},if=none,cache=writeback,discard=ignore,format=qcow2,id=virtio-drive"], ["-device", "virtio-scsi"], ["-drive", "file=${var.cloud_init_image},format=raw,if=none,id=c1"], ["-device", "scsi-cd,drive=c1"], ["-m", "${var.memory}"], ["-smp", "cpus=${var.cpus}"], ["-serial", "mon:stdio"]]
+  qemuargs          = [["-m", "${var.memory}"], ["-smp", "cpus=${var.cpus}"], ["-cdrom", "${var.cloud_init_image}"], ["-serial", "mon:stdio"]]
```

Comment:
I see. The existing qemuargs doesn't work with uefi and I want to keep the options same for all the distros.
Most likely this will need few iterations to arrive at the options which work for all distros including uefi.